src/exodus/tx.cpp

```diff
@@ -875,6 +878,59 @@ bool CMPTransaction::interpret_SimpleMint()
     return true;
 }
 
+/** Tx 1026 */
```

Comment:
Need to update comment due to we will remove duplicated transaction type.

Reply:
nice catch